docs/offer/offer.yaml

```diff
@@ -310,3 +310,59 @@ paths:
                 status: 404
                 code: OFFER_NOT_FOUND
                 message: Offer with provided data was not found.
+  /offers/price-range{?authorRole}:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Offers
+      summary: Find and return min and mix price.
+      description: Find and return min and mix price for specific role.
+      produces:
+        - application/json
+      parameters:
+        - name: query
+          in: path
+          required: true
+          description: The query with the role "?authorRole=tutor" or "?authorRole=student"
+          type: string
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/offer'
+              example:
+                minPrice: 300
+                minPrice: 500
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 400
+                code: INVALID_ID
+                message: ID is invalid.
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 401
+                code: UNAUTHORIZED
+                message: The requested URL requires user authorization.
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 404
+                code: OFFER_NOT_FOUND
+                message: Offer with provided data was not found.
```

Comment:
doesn't look good

you must show exactly what you gonna receive if you provide wrong data to retrieve this info
1) offer_not_found doesn't have THAT message
2) think about what error it can cast